### DISCUSSION TITLE
我的 Linux 开发机 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#109] [Comments]
# 我的 Linux 开发机 | 夜天之书

首先说一下结论：最终我选择了基于 Arch Linux 的 Garuda Linux 发行版作为基础来搭建自己的 Linux 开发机。

https://www.tisonkun.org/2023/02/11/garudalinux/

**Comment by Goooler** [comment]
上大学时候我也用 Source Code Pro，自从出了 JetBrains Mono 之后就没切回去过

**Reply by tisonkun** [reply]
哈哈，有什么主要的字母区别么。我记得比较喜欢 Source Code Pro 是它等宽，所以码代码的时候会很舒服。不过写作的时候确实就有点太稀疏了，我会用 Times New Roman 之类的字体。

**Reply by Goooler** [reply]
带 mono 后缀的都是等宽字体，都可以拿来做代码字体。JBM 自带连字，这个开了之后看起来很舒服，IDEA 默认的字体现在就是，可以开连字感受一下。

**Reply by tisonkun** [reply]
哈哈，我可能大部分时候不喜欢连字，已经习惯了 `->`.

**Reply by tisonkun** [reply]
而且目前连字的 display 效果我并不喜欢，如果有一天我发现很好看，可能会换。

**Comment by eagleoflqj** [comment]
Hi，要不要体验一下 macOS 的[小企鹅](https://github.com/fcitx-contrib/fcitx5-macos-installer/blob/master/README.zh-CN.md)输入法和rime插件，和Linux的配置基本一样

**Comment by houkensjtu** [comment]
> 人不 Arch 枉少年
人不 Gentoo 枉中年
